deps/build.jl

```diff
@@ -65,8 +65,13 @@ MPI_constants = [
     # proc
     "MPI_PROC_NULL",
 
+    # memory layout
+    "MPI_ORDER_C",
+    "MPI_ORDER_FORTRAN",
```

Comment:
these constants/functions need to be added to `src/win_mpiconstants.jl` as well. They're available from https://github.com/microsoft/Microsoft-MPI/blob/182a2815f6ce2dc4efc2f00b2bd8eb5e0f34ff7f/src/include/mpif.h

Reply:
I have added those in a previous commit.